assets/css/_sass/base/_typography.scss

```diff
@@ -0,0 +1,147 @@
+.list {
+  ol {
+    padding: 0 0 0 1em;
+    margin: 0;
+  }
+
+  li {
+    list-style: none;
+    padding-bottom: 0.5em;
+
+    &.subterm {
+      padding-left: pxToRem(10);
+    }
+
+    &:last-child {
+      padding-bottom: 0;
+    }
+
+    &.subterm:before {
+      content: '\21B3';
+      padding-right: pxToRem(5);
+    }
+  }
+}
+
+.list-semicolon {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  & > li {
+    display: inline;
+
+    &:not(:last-child)::after {
+      content: '; ';
+    }
+  }
+}
+
```

Comment:
I think the list definitions should be in `components/_lists.scss` rather than typography